enclave/cert.c

```diff
@@ -643,11 +681,11 @@ oe_result_t oe_cert_chain_read_pem(
         OE_RAISE(OE_OUT_OF_MEMORY);
```

Comment:
Nice if you can print the message.

Reply:
OE_RAISE already print error message by default

Reply:
ok